xml/en/docs/http/ngx_http_slice_module.xml

```diff
@@ -29,6 +29,23 @@ configuration parameter.
 </section>
 
 
+<section id="issues" name="Known Issues">
+
+<para>
+A slice cannot be updated with background subrequests enabled,
+such as
+<link doc="ngx_http_proxy_module.xml" id="proxy_cache_background_update"/>,
+<link doc="ngx_http_ssi_module.xml">SSI</link>,
+<link doc="ngx_http_perl_module.xml">Perl</link>, or
+<link doc="../njs/reference.xml" id="r_subrequest">njs subrequests</link>.
+In these cases, the value of the
+<link id="var_slice_range"><var>$slice_range</var></link> variable
+will be an empty string and the entire cache item will be updated.
+</para>
+
+</section>
+
+
```

Comment:
What about this text:
```
<para>
The module doesn't currently work together with
<link doc="ngx_http_proxy_module.xml" id="proxy_cache_background_update">
background cache update</link>,
as well as with subrequests in
<link doc="ngx_http_ssi_module.xml">SSI</link>,
<link doc="ngx_http_perl_module.xml">Perl</link>, or
<link doc="../njs/reference.xml" id="r_subrequest">njs</link> modules,
because it is not possible to construct a range request
in a nested subrequest.
</para>
```

Reply:
Thanks for the good variant - updated and forcepushed for review

Reply:
More thoughts after f2f discussion with @arut:
- it makes sense to mention what actually happens in this case;
- just one (most relevant) example of using slice in subrequests.
```
Currently, the module doesn't work as expected in subrequests such as
<link doc="ngx_http_proxy_module.xml" id="proxy_cache_background_update">
background cache update</link>.
In this case, a request is constructed without byte-range support.
```